config/default.yml

```diff
@@ -3347,9 +3347,15 @@ Style/MethodCallWithArgsParentheses:
   StyleGuide: '#method-invocation-parens'
   Enabled: false
   VersionAdded: '0.47'
-  VersionChanged: '0.48'
+  VersionChanged: '0.61'
   IgnoreMacros: true
   IgnoredMethods: []
+  AllowParenthesesInMultilineCall: false
+  AllowParenthesesInChaining: false
+  EnforcedStyle: require_parentheses
+  SupportedStyles:
+    - require_parentheses
+    - require_no_parentheses
```

Comment:
I'd name this `omit_parentheses` and the change terminology you used in the code as well to match it. When there's a word that exactly conveys the meaning we're after, using it is also preferable over using negation.

Reply:
I had those named `require` and `omit`, but saw the `Style/MethodDefParentheses`, which uses the names above, so I got them for "symmetry". Anyway, I can rename them here.

Reply:
Yeah, we've been pretty inconsistent with some naming. At some point we'll have to address this, but now for now `omit` is fine.